trunk/SUAVE/Methods/Weights/Buildups/eHelicopter/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""SUAVE.Methods.Weights.Correlations.eHelicopter
```

Comment:
Use verbose names for classes and functions.

Reply:
Using this to ask generally about verbose names - would "electricHelicopter", etc. be sufficiently verbose?

Reply:
I'm gonna say yes.

Reply:
Also, our naming convention in SUAVE would be "Electric_Helicopter" for the folder, and "electric_helicopter" if this is the method name.

Reply:
Changed to comply with naming convention